shanoir-ng-anonymization/pom.xml

```diff
@@ -25,6 +25,8 @@ along with this program. If not, see https://www.gnu.org/licenses/gpl-3.0.html
 	<url>http://maven.apache.org</url>
 
 	<properties>
+		<maven.compiler.source>1.6</maven.compiler.source>
```

Comment:
Why do you change this ?

Reply:
I guess it's an error, I just removed it.
I probably mistakenly added this when I had a different version of maven or used a different build command.